packages/backend/package.json

```diff
@@ -29,11 +29,11 @@
     "express-graphql": "0.12.0",
     "graphql": "15.5.0",
     "graphql-request": "3.4.0",
-    "graphql-tag": "2.11.0",
+    "graphql-tag": "2.12.4",
     "graphql-tools": "7.0.4",
     "imgix-core-js": "2.3.2",
     "node-fetch": "2.6.1",
-    "sourcecred": "0.8.4",
+    "sourcecred": "0.8.5",
```

Comment:
make sure you test this, we've encountered a number of breaking changes in recent sourcecred updates

Reply:
yes, it mostly adds functionality keeping the old stuff the same.